### PR TITLE
ESX Version

### PR DESCRIPTION
**Pull request by FripeX**
Added ESX Version

### CODE REVIEW

File: lua/client.lua [hunk 1]
```diff
@@ -7,10 +6,16 @@ ActivePosters = {}
 CurrentPoster = {}
 GLOBAL_COORDS = nil
 TXD = nil 
-local QBCore = exports['qb-core']:GetCoreObject()
+local ESX = nil
```
Comment (Mathu-lmn) [inline lua/client.lua:9]: I'd suggest adding a configuration option instead of deleting this QB support

File: lua/client.lua [hunk 2]
```diff
@@ -20,13 +25,14 @@ AddEventHandler('onResourceStart', function(resource)
     end
 end)
 
-RegisterNetEvent("QBCore:Client:OnPlayerUnload", function()
+RegisterNetEvent('esx:onPlayerLogout')
+AddEventHandler('esx:onPlayerLogout', function()
```
Comment (Mathu-lmn) [inline lua/client.lua:29]: This is an unneeded change....

File: lua/client.lua [hunk 3]
```diff
@@ -165,7 +199,7 @@ function DeleteImage()
                         disable = {car = true, move = true, combat = true},
                         anim = { dict = 'mini@repair', clip = 'fixing_a_ped' },
                     }) then
-                        TriggerServerEvent("posters:deleteImage", currentKey, PlayerData.citizenid == currentPoster.cid)
+                        TriggerServerEvent("posters:deleteImage", currentKey, PlayerData.identifier == currentPoster.cid)
```
Comment (Mathu-lmn) [inline lua/client.lua:202]: Same goes here, use the config option !

**Review by Mathu-lmn — CHANGES_REQUESTED** [review]
Please don't overwrite the existing script / code, but make it a configuration instead !

File: lua/client.lua (hunk 1 above)
Comment (FripeX) [reply lua/client.lua]: added

**Comment by Mathu-lmn** [comment]
Readme got updated, might be worth checking if it fits your needs

**Comment by Mathu-lmn** [comment]
Also might want to set default framework to qb

**Comment by FripeX** [comment]
> Anyone validate this works before I merge?

All good, you can just check the config if you want to set the default framework to qb

**Pull request closed without merging**